jsk_arc2017_baxter/euslisp/lib/arc-interface.l

```diff
@@ -361,9 +361,9 @@
       (send *baxter* :reset-pose arm)
       (send *baxter* :rotate-gripper arm gripper-angle :relative nil)
       (setq offset
-            (cond ((eq bin :a) (float-vector 0 offset-y 250))
-                  ((eq bin :b) (float-vector 0 offset-y 200))
-                  ((eq bin :c) (float-vector 0 offset-y 200))))
+            (cond ((eq bin :a) (float-vector -100 offset-y 250))
+                  ((eq bin :b) (float-vector -200 offset-y 200))
+                  ((eq bin :c) (float-vector -150 offset-y 200))))
```

Comment:
this value is modified because i change to use `:ik->bin-center`.
It doesn't work?